documents/employment-contract-no.md

```diff
@@ -0,0 +1,128 @@
+# Ansettelseskontrakt
+
+## Arbeidsområde
+
+Arbeidets art er beskrevet i den ansattes rollebeskrivelse. Arbeidet skal utøves
+i samsvar med gjeldende formål og den til enhver tid vedtatt strategi for,
```

Comment:
"vedtatt strategi for, "

Mangler noe etter for, eller vi kan ta vekk for. 

Slik at den er: " ... tid vedtatte strategi..."